serveradmin/api/urls.py

```diff
@@ -6,6 +6,7 @@
 from django.conf.urls import url
 
 from serveradmin.api.views import (
+    health_check,
```

Comment:
Is Health Check really needed for this PR?

Reply:
Yes, as the new setup contains loadbalanced webservers which we didn't have before.